netlify-functions-ecommerce/netlify/functions/createOrder.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const stripe = require('../../integrations/stripe');
+const { Cart, Order } = require('../../models');
+const connect = require('../../connect');
+
+const handler = async(event) => {
+  try {
+    event.body = JSON.parse(event.body || {});
```

Comment:
Do you need to do anything to configure Stripe to use this endpoint? Just want to know for readme purposes

Reply:
> Do you need to do anything to configure Stripe to use this endpoint? Just want to know for readme purposes

Well currently, when checkout is completed, on the success page it fires an api request to create the order. I'm sure that you could set up a webhook so that the user wouldn't have to wait for the success page and instead the webhook would trigger this route.